growth/too/models.py

```diff
@@ -68,7 +68,9 @@ def create_all():
             'filterScheduleType': 'block',
             'airmass': 2.5,
             'schedule_strategy': 'tiling',
-            'mindiff': 30.*60.
+            'mindiff': 30.*60.,
+            'doMaxTiles': False,
```

Comment:
Why do you need both a `doMaxTiles` and a `max_nb_tiles` argument?

Reply:
it mimics the check box and slider. in principle it could be negative value = not use and positive = use, but I think it is more clear with the two arguments.

Reply:
It's more clear if it imposes the maximum tile limit if this keyword argument is present, and imposes no limit if it is absent.

Reply:
I don't think I agree. To me, this looks like a configuration file (of sorts), and all of the defaults are explicitly here?

Reply:
gwemopt is your package, not mine, so this is your choice. This is just my advice.

Reply:
its a point well taken, this is how it is done now, can maybe think about moving that direction in a future release.